tests/template_tests/test_project_generation.py

```diff
@@ -33,6 +36,15 @@ def test_template_with_extra_answers(copie): # codespell:ignore copie
 
     assert result.exit_code == 0, f"Exited with code {result.exit_code}, expected 0"
     assert result.exception is None, result.exception
-    assert result.project_dir.is_dir(), f"Project directory {result.project_path} not found"
-    with open(result.project_dir / "README.md") as f:
+    assert result.project_dir.is_dir(), f"Project directory {result.project_dir} not found"
+    with open(result.project_dir / extra_context["project_name"] / "README.md") as f:
         assert f.readline() == f"# {extra_context['project_name']}\n", f"{f.readline()} is not the same as {extra_context['project_name']}\n"
+
+def test_cli():
+    """
+    Testing if the CLI works and returns a successful exit code on execution
+    """
+    os.mkdir("testcli")
+    return_code = subprocess.run(["pybamm-cookiecutter", "--defaults"], cwd = "./testcli")
+    shutil.rmtree("testcli")
```

Comment:
Files and folders of the generated project are being checked above so this minimal test should be fine ✅